src/collective/sidebar/browser/sidebar.py

```diff
@@ -455,19 +456,14 @@ def get_action_icon(id1):
     """
     Returns icons for action ids
     """
-    icon_map = {
-        'cut': 'glyphicon glyphicon-scissors',
-        'copy': 'glyphicon glyphicon-duplicate',
-        'paste': 'glyphicon glyphicon-open-file',
-        'delete': 'glyphicon glyphicon-trash',
-        'rename': 'glyphicon glyphicon-random',
-        'ical_import_enable': 'glyphicon glyphicon-calendar',
-        'ical_import_disable': 'glyphicon glyphicon-calendar',
-    }
-    if id1 and id1 in icon_map:
```

Comment:
`id1` if referring to the index of the item in the loop should be named `idx`.

Reply:
Renamed to more readable "action_id"